src/main.js

```diff
@@ -24,12 +26,13 @@ k.scene('forest_junction', (enter_tag) =>
 k.scene('campus_house_1', (enter_tag) =>
     bootstrap(campusHouse1, { enter_tag })
 );
+k.scene('fishing', (enter_tag) => miniGameBootstrap(fishing, { enter_tag }));
 
 // Game Machine Scenes
 k.scene('startScreen', gameStartScreen);
 k.scene('lose', loseScreen);
 
-k.go('start');
+k.go('forest', 'spawn_bottom');
```

Comment:
can you revert please to start scene again?

Reply:
I will fix this issue once I have finally fixed the game. If you can, try to remind me lol